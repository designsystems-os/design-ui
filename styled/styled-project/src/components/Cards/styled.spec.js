import React from 'react';
import TestRenderer from 'react-test-renderer';
import 'jest-styled-components';

import Card from './styled';

////////////////////////////////////////
/* Your Styled Card Tests Go Here...  */
////////////////////////////////////////

/*
 * Default Styles
 */
test('it renders default styles', () => {
  const component = TestRenderer.create(<Card />).toJSON();
  expect(component).toMatchSnapshot();
  expect(component).toHaveStyleRule('display', 'flex');
});

/*
 * Custom Styles
 */
test('it renders custom styles', () => {
  const component = TestRenderer.create(<Card display="grid" />).toJSON();
  expect(component).toMatchSnapshot();
  expect(component).toHaveStyleRule('display', 'grid');
});
