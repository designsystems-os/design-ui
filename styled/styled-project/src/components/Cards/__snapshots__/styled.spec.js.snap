// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders custom styles 1`] = `
.c0 {
  display: grid;
  height: 100px;
  width: 100px;
  border-radius: 0;
}

<div
  className="c0"
  display="grid"
  height="100px"
  width="100px"
/>
`;

exports[`it renders default styles 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100px;
  width: 100px;
  border-radius: 0;
}

<div
  className="c0"
  display="flex"
  height="100px"
  width="100px"
/>
`;
